lib/matplotlib/backends/backend_ps.py

```diff
@@ -665,8 +665,9 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
                 curr_stream[1].append(
                     (item.x, item.ft_object.get_glyph_name(item.glyph_idx))
                 )
-            # append the last entry
-            stream.append(curr_stream)
+            # append the last entry if exists
+            if curr_stream:
```

Comment:
On consideration, I think the looser check here is better than what I had because I suspect other falsy things (like empty lists) would also break the consumer code.